### PR TITLE
Version 1.1, Odoo 11 support

### LINKED ISSUE #16: Odoo 11 support (including Python 3)
- [x] python 3 support (522d255ede683c06e552da8a419326ee80f65c4e, this solution works with python 3 only: the only solution that would work with both requires proper implementation of namespace packages, ie emptying `odoo/__init__.py`)
- [x] fix #10 
  - [x] Odoo patch (https://github.com/acsone/odoo/commit/b43e13e1e8bf6c3e647814771d11e37ab15bc81f)
  - [x] update `setuptools-odoo` core
  - [x] update `setuptools-odoo-make-default`
- [ ] update list of core addons
- [x] add some v11 addons in test data
- [x] update documentation

### PR DESCRIPTION
**Pull request by sbidoul**
Requires either https://github.com/odoo/odoo/pull/19517 (python 3 only) or odoo-autodiscover 2.0 (for python 2 and 3).

fixes #16

### CODE REVIEW

File: setuptools_odoo/core.py [hunk 1]
```diff
@@ -12,50 +12,68 @@
...
     '10.0': {
         'odoo_dep': 'odoo>=10.0,<10.1dev',
         'base_addons': base_addons.odoo10,
-        'addon_dep_version': '',
         'pkg_name_pfx': 'odoo10-addon-',
         'addons_ns': 'odoo.addons',
+        'namespace_packages': ['odoo', 'odoo.addons'],
+    },
+    '11.0': {
+        'odoo_dep': 'odoo>=11.0a,<10.1dev',
```
Comment (Fenkiou) [inline setuptools_odoo/core.py]: Maybe the dev version number isn't available yet but 10.1dev isn't correct here.

Comment (sbidoul) [reply setuptools_odoo/core.py]: Indeed. Thanks for reporting.

Note this is still very experimental as we've not agreed yet with Odoo on how best to support this.

Comment (sbidoul) [reply setuptools_odoo/core.py]: Fixed and rebased.

**Comment by sbidoul** [comment]
https://github.com/odoo/odoo/pull/19517, so let's merge this one.

**Pull request merged**